docs/integrations/enterprise-connectors/destination-salesforce.md

```diff
@@ -5,12 +5,29 @@ dockerRepository: airbyte/destination-salesforce
 
 ## Overview
 
-This page guides you through the process of setting up the [Salesforce](https://www.salesforce.com/) destination connector.
+The Salesforce destination connector enables [data activation](/platform/next/move-data/elt-data-activation) by syncing data from your data warehouse to Salesforce objects. This connector is designed for operational workflows where you need to deliver modeled data directly to your CRM for sales, marketing, and customer success teams.
+
+:::note
+This connector requires Airbyte version 1.8 or higher.
+:::
+
+The connector uses the [Salesforce Bulk API v62.0](https://developer.salesforce.com/docs/atlas.en-us.api_asynch.meta/api_asynch/asynch_api_intro.htm) for efficient data loading and supports OAuth 2.0 authentication with comprehensive error handling through [rejected records](/platform/next/move-data/rejected-records) functionality.
+
+### Key Features
+
+- **Data Activation Support**: Sync enriched customer data, lead scores, and product usage metrics from your warehouse to Salesforce
+- **Bulk API Integration**: Uses Salesforce Bulk API v62.0 for high-performance data loading with 100MB batch processing
+- **Rejected Records**: Failed records are automatically captured and stored in S3 for analysis and reprocessing
+- **OAuth 2.0 Authentication**: Secure authentication with support for both sandbox and production environments
+- **CSV Format Processing**: Data is processed in CSV format for optimal compatibility with Salesforce Bulk API
+
 
 ## Prerequisites
 
-- [Salesforce Account](https://login.salesforce.com/) with Enterprise access or API quota purchased
-- (Optional, Recommended) Dedicated Salesforce [user](https://help.salesforce.com/s/articleView?id=adding_new_users.htm&type=5&language=en_US)
+- [Salesforce Account](https://login.salesforce.com/) with Enterprise edition or Professional edition with API access purchased
```

Comment:
There will be a merge conflict with https://github.com/airbytehq/airbyte/pull/64515/files#diff-510b0ea3020eb0010ada50c543af6a2328f1f5154f474d5c938005d6ffe3d4b3R13